README.md

```diff
@@ -342,15 +344,22 @@ var emeOptions = {
   }
 };
 
-player.eme.initializeMediaKeys(emeOptions, function(error) {
+var emeCallback = function(error) {
   if (error) {
     // do something with error
   }
 
   // do something else
-});
+};
+
+player.eme.initializeMediaKeys(emeOptions, emeCallback, suppressErrorsIfPossible);
 ```
 
+With `suppressErrorsIfPossible` set to `false` (the default), the error handler
+will be invoked after the callback finishes, and `error` called on the player.
+When set to `true`, the error handler will not be invoked, with the exception
+of `mskeyerror` errors in IE11.
```

Comment:
Probably would be good to explain why it doesn't happen in IE11 (at least, just that the error can't be suppressed there).

Reply:
Done!